lms/templates/instructor/instructor_dashboard_2/e-commerce.html

```diff
@@ -255,13 +256,13 @@ <h3 class="error-msgs" id="error-msg"></h3>
          }
          if($('#invoice_number').val() == "") {
                      $('#error-msg').attr('class','error-msgs')
-                     $('#error-msg').html("${_('The Invoice Number field cannot be empty.')}").show();
+                     $('#error-msg').html("${_('The Invoice Number field cannot be empty.') | n, h, js_escaped_string}").show();
```

Comment:
Sounds like an ideal solutions would be to move all of this Javascript to a js file. Sounds like we need a "Put Javascript Where It Belongs" Bash next. :)